openapi/propolis-server.json

```diff
@@ -436,18 +436,6 @@
     }
   },
   "components": {
-    "responses": {
```

Comment:
Since the `openapiv3` bump is causing this change, can you regenerate the file for `propolis-server-falcon.json` as well? (build `propolis-server` on illumos with `falcon` feature enabled, and have it generate the document)

Reply:
Done in e919928, and updated the OpenAPI test to compare against the right base document using the enabled features.